tests/functional/monitoring/conftest.py

```
@@ -1181,25 +1184,51 @@ def measure_change_client_ocs_version_and_stop_heartbeat(
+        additional_wait_time = 60 * 3  # 3 minutes
+        logger.info(
+            f"Waiting additional {additional_wait_time} seconds with incompatible version"
+        )
+        time.sleep(additional_wait_time)
         return
 
     def teardown():
         nonlocal client
         nonlocal client_cluster
+        nonlocal current_version
         logger.info(f"Switch to client cluster ({client_cluster})")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)